### DISCUSSION TITLE
Error getting mesh device id

### DISCUSSION THREAD

**Discussion by tpozzi** [amidaware/tacticalrmm#1412] [General]
Hi guys,
My name is Thadeu and I´m kind of stuck since I upgraded the version of my Tactical RMM
I ran the command below and got these result.
Can some one help me please
 
> root@patch:/rmm/api/tacticalrmm# /rmm/api/env/bin/python /rmm/api/tacticalrmm/manage.py check_mesh
> Mesh site: https://mesh.server.nl
> Mesh username: kbppnaua
> Mesh token: tokenkeything
> Mesh device group: TacticalRMM
> Auth token ok:
> tokenkeything
> Mesh url ok:
> wss://mesh.server.nl/control.ashx?auth=tokenkeything
> Error getting mesh device id:
> received 1005 (no status code [internal]); then sent 1005 (no status code [internal])

**Comment by wh1te909** [comment]
first of all you should never be running those commands as root
where is your mesh server running? on same box as trmm or external?
did you make any changes in mesh? that error usually means trmm and mesh are out of sync which only happens if you manually changed something in mesh, like the username or device group or something

**Reply by wh1te909** [reply]
and please confirm your mesh is working as in you are able to browse to `https://mesh.yourdomain.com` and you see the login screen

**Comment by tpozzi** [comment]
Hi Dan.  I migrated from one VPS to another and then I got this error.  both rmm and mesh are running on the same Box.  not that it matters but just to clarify I already ran the update --force to adjust all permissions

Obter o Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by tpozzi** [comment]
Yes is working, I can open it in a browser

Obter o Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by tpozzi** [comment]
Any suggestion?!?!??!

Obter o Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by dinger1986** [reply]
Presumably you haven't read the rest of @wh1te909 suggestions above and that something must have changed?

**Reply by tpozzi** [reply]
Dear [@dinger1986](https://github.com/dinger1986),

I read all coments from [@wh1te909](https://github.com/wh1te909)<https://github.com/wh1te909>  and all he did was some questions. And I ansewred it all.

I did install as root but ran update.sh  --force as normal user to right my mistake and solve permissions issue.

My TRMM and mesh central  are runing in the same box.

And finaly I do have access to mesh web page  and can access with my creds

So as I told you, I already ansewred but I need some help

Could you or somebody help me please

Kind regards,
Thadeu

**Reply by wh1te909** [reply]
you can open a ticket here and we can login to your server and check it: https://docs.tacticalrmm.com/support/#commercial-support

**Reply by tpozzi** [reply]
Dear @wh1te909 unfortunately that is not an option unce my company is brand new and I have no Money yet. I got my first client  and wanted to continue to use this RMM without neeed to reinstall it  all

**Reply by wh1te909** [reply]
if you are willing to give us unattended root ssh access then I will diagnose the issue for free. we don't charge for diagnosis. and if the fix is very simple we'll fix it for free.

**Reply by tpozzi** [reply]
sorry can't do it. Too risk. With all do respetc I don't know what you will do. But if i give access in my computer for you do the diagnostics, than I'm ok with that

**Reply by wh1te909** [reply]
no problem, I understand. If you would like us to meet and control your computer that will not be free and will be paid support.
